dbms/src/Common/FailPoint.cpp

```diff
@@ -200,6 +222,40 @@ void FailPointHelper::wait(const String & fail_point_name)
         ptr->wait();
     }
 }
+
+void FailPointHelper::initRandomFailPoints(Poco::Util::LayeredConfiguration & config, Poco::Logger * log)
+{
+    String random_fail_point_cfg = config.getString("flash.random_fail_points", "");
```

Comment:
are there some test plans? how about open failpoint for ci test

Reply:
Plan to add a regular cluster test, since some failpoints need to be test under parrel workloads.